easybuild/easyconfigs/j/JUBE/JUBE-2.6.1.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonBundle'
+
+name = 'JUBE'
+version = '2.6.1'
+
+homepage = 'https://www.fz-juelich.de/jsc/jube'
+description = """The JUBE benchmarking environment provides a script based
+framework to easily create benchmark sets, run those sets on different
+computer systems and evaluate the results.
+"""
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+use_pip = True
+
+exts_list = [
+    ('PyYAML', '6.0.1', {
+        'checksums': ['bfdf460b1736c775f2ba9f6a92bca30bc2095067b8a9d77876d1fad6cc3b4a43'],
+        'modulename': 'yaml',
+    }),
+    ('jube2', version, {
+        'source_urls': ['https://apps.fz-juelich.de/jsc/jube/jube2/download.php?file='],
+        'sources': ['%(name)s-%(version)s.tar.gz'],
+        'checksums': ['348ebeb4db0ea56889a5d13bf49e394fccc0992f620e8c0c61c40185db13ad64'],
+        'modulename': 'jube2',
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/jube'],
+    'dirs': [],
+}
+
+sanity_check_commands = ["jube --version"]
+
+sanity_pip_check = True
+
+modextrapaths = {
+    'JUBE_INCLUDE_PATH': 'share/jube/platform/slurm'
+}
+
+modluafooter = 'execute {cmd=\'eval "$(jube complete)"\',modeA={"load"}}'
```

Comment:
If there is a modluafooter then there should also be a modtclfooter...
Is it doable in this case?